### PR TITLE
Support color profiles strip

### PR DESCRIPTION
**Pull request by ReDetection**
If you don't do that and you have large profile embedded, you risk to have 99% of image storage to waste on color profiles. This option can cut 2MB files down to 20KB.

### CODE REVIEW

**Comment by Magomogo** [comment]
Thanks

**Comment by ReDetection** [comment]
do you plan to release a new version?

File: test/ConverterTest.php [hunk 1]
```diff
@@ -4,6 +4,22 @@
 
 class ConverterTest extends \PHPUnit_Framework_TestCase
 {
+    public function testRemovesColorProfileInformation()
+    {
+        $bin = self::converter()->convert(file_get_contents(__DIR__ . '/data/colorProfile.jpeg'), self::command('strip'));
+        $tmpFile = self::tmpDir() . 'profilesCheckRemove.jpg';
+        file_put_contents($tmpFile, $bin);
```
Comment (kevich) [inline test/ConverterTest.php]: Why you don't delete temp files after usage?

Comment (ReDetection) [reply test/ConverterTest.php]: fixed in #13

**Pull request merged**